<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>intoWindow</class>
 <widget class="QMainWindow" name="intoWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>540</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>960</width>
    <height>540</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>960</width>
    <height>540</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0.5, x2:1, y2:0.5, stop:0.125 rgba(0, 0, 0, 255), stop:1 rgba(44, 183, 255, 255));</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="Line" name="frameRight">
    <property name="geometry">
     <rect>
      <x>353</x>
      <y>0</y>
      <width>607</width>
      <height>540</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 7px solid;
border-color: rgb(0, 0, 0);
background-color: rgba(0, 0, 0, 0);
border-radius: 1em;</string>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="frameLeft">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>360</width>
      <height>540</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 7px solid;
border-color: rgb(0, 0, 0);
background-color: rgba(0, 0, 0, 0);
border-radius: 1em;</string>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="intoLogo">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>300</width>
      <height>300</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(0, 0, 0, 0);
border: 1px solid rgb(255, 255, 255);
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>Images/IntoTheSpiderVerse.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="intoText">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>335</y>
      <width>360</width>
      <height>45</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lucida Console</family>
      <pointsize>28</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(0, 0, 0, 0);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Into The</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
    <property name="indent">
     <number>-1</number>
    </property>
   </widget>
   <widget class="Line" name="frameLeftBack">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>360</width>
      <height>540</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 7px solid;
border-color: rgb(0, 0, 0);
background-color: rgba(0, 0, 0, 0);</string>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="frameRightBack">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>0</y>
      <width>600</width>
      <height>540</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-right: 7px solid;
border-top: 7px solid;
border-bottom: 7px solid;
border-color: rgb(0, 0, 0);
background-color: rgba(0, 0, 0, 0);</string>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QListWidget" name="listOfMusic">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>30</y>
      <width>550</width>
      <height>300</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>14</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(0, 0, 0, 0);
border: 0px;
color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QLabel" name="nowPlayingText">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>345</y>
      <width>550</width>
      <height>45</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>24</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(0, 0, 0, 0);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Now Playing:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QSlider" name="musicSlider">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>450</y>
      <width>550</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QSlider {
    background: transparent;
}
QSlider::groove:horizontal {
    height: 2px;
    background: rgb(255, 255, 255);
}
QSlider::handle:horizontal {
    background: #000000;
    border: 3px solid #000000;
    width: 2px;
    margin: -12px 0; 
}
</string>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="playButton">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>420</y>
      <width>75</width>
      <height>75</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(0, 0, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>Images/playbutton.png</normaloff>Images/playbutton.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>75</width>
      <height>75</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pauseButton">
    <property name="geometry">
     <rect>
      <x>185</x>
      <y>420</y>
      <width>75</width>
      <height>75</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(0, 0, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>Images/pausebutton.png</normaloff>Images/pausebutton.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>75</width>
      <height>75</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="nextButton">
    <property name="geometry">
     <rect>
      <x>265</x>
      <y>420</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(0, 0, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>Images/nextbuttonw.png</normaloff>Images/nextbuttonw.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>80</width>
      <height>80</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="prevButton">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>420</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(0, 0, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>Images/prevbuttonw.png</normaloff>Images/prevbuttonw.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>80</width>
      <height>80</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="statusOfMusicText">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>390</y>
      <width>550</width>
      <height>45</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>24</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(0, 0, 0, 0);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="spiderText">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>375</y>
      <width>360</width>
      <height>45</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lucida Console</family>
      <pointsize>28</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(0, 0, 0, 0);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Spider-Verse</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <zorder>frameLeftBack</zorder>
   <zorder>frameRight</zorder>
   <zorder>frameLeft</zorder>
   <zorder>intoLogo</zorder>
   <zorder>intoText</zorder>
   <zorder>frameRightBack</zorder>
   <zorder>listOfMusic</zorder>
   <zorder>nowPlayingText</zorder>
   <zorder>musicSlider</zorder>
   <zorder>playButton</zorder>
   <zorder>pauseButton</zorder>
   <zorder>nextButton</zorder>
   <zorder>prevButton</zorder>
   <zorder>statusOfMusicText</zorder>
   <zorder>spiderText</zorder>
  </widget>
 </widget>
 <resources>
  <include location="Images/images.qrc"/>
 </resources>
 <connections/>
</ui>
